main.js

```diff
@@ -0,0 +1,17 @@
+const addButton = document.getElementById('add');
+const itemsContainer = document.getElementById('itemsContainer');
+const inputField = document.getElementById('inputField');
+
+addButton.addEventListener('click', () => {
+  const paragraph = document.createElement('p');
+  paragraph.classList.add('p-style');
+  paragraph.innerText = inputField.value;
+  itemsContainer.appendChild(paragraph);
```

Comment:
- [ ] When I refresh the page after adding some to-do tasks, boom! all tasks disappeared :angry: . The reason is that our tasks(data) is not saved across browser sessions ([localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)). In the requirements of this project, we are equally asked to make use of this technology. You may require to refactor your code to use an array of items. Please let's try it out, it will be fun!.